docs/features/wait/multi.md

```diff
@@ -0,0 +1,20 @@
+# Multi Wait strategy
```

Comment:
Called `WaitAllStrategy` in Java.

Reply:
Indeed. The function call is `wait.ForAll`, which is aligned at the API level. But saying `All` in the docs is less clear imho that multi. But I'm open to rename it

Reply:
I think both are fine:
* We wait for _all_ wait strategies
* We wait for _multiple_ wait strategies

In order to align with Java, I'd be happy if we consider the rename here as well.

Reply:
As discussed, we are going to rename the `multi.go` file to `all.go`:
- consistency with Java
- the exposed API is not changed (wait.ForAll)